NuGet.config

```diff
@@ -17,6 +17,7 @@
     <add key="aspnet-extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
     <add key="dotnet-windowsdesktop" value="https://dotnetfeed.blob.core.windows.net/dotnet-windowsdesktop/index.json" />
     <add key="dotnet-toolset" value="https://dotnetfeed.blob.core.windows.net/dotnet-toolset/index.json" />
+    <add key="NetCoreInstaller" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-installer-temp/nuget/v3/index.json" />
```

Comment:
dotnet-installer-temp?

Is this going to flow via darc/maestro?

Reply:
Eventually yes, once I've figured out the logistics of building this using Arcade. I'll probably start looking into that later this week or early next. Right now, we have a private repo and I have a CI machine I can use to build/sign. The bootstrapper is quite different from the typical builds we have in Arcade and I figured it was better to spend time on getting the last bugs sorted in the engine than making this run on Arcade.